lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.html

```diff
@@ -6,6 +6,7 @@
                 [errorStateMatcher]="matcher"
                 placeholder="{{ 'SEARCH.FILTER.RANGE.FROM-DATE' | translate }}"
                 [attr.aria-label]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
+                [attr.title]="'SEARCH.FILTER.RANGE.FROM-DATE' | translate"
```

Comment:
I don't think this fix resolves the issue, as described here https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title -> "While title can be used to provide a programmatically associated label for an input element, this is not good practice. Use a label instead." So in my opinion this isn't a valid fix for this issue